device-telemetry/app/com/microsoft/azure/iotsolutions/devicetelemetry/services/Rules.java

```diff
@@ -456,4 +461,56 @@ private RuleServiceModel getServiceModelFromJson(JsonNode response) {
                     "Could not parse data from Key Value Storage"));
         }
     }
-}
+
+    private CompletionStage<Integer> getRuleCountAsync() {
+        return this.prepareRequest(null)
```

Comment:
nit: prepareRequest don't have parameter.

Reply:
it has a parameter of id